scripts/extract-l10n.sh

```diff
@@ -117,7 +108,7 @@ fi
 ./scripts/run_l10n_extraction.sh
 
 if [[ $DRY_RUN == false ]]; then
-  commit
+  ./scripts/push_l10n_extraction.sh
```

Comment:
the new .sh doesn't just replace what `commit` does - it also replaces part of what `create_pull_request` does, so that needs adjusting too.

Reply:
Good catch. Seems the typical flow requires the user to push manually anyway, but I removed the line which is pushing and only kept the line that is creating the PR.

I could actually just remove that script in this PR just didn't want to do it yet until we know the new flow is working well.

Reply:
So have you removed the script? Or you're going to remove it before you merge? Or 😕 ...

Reply:
I removed the git push on line 103

https://github.com/mozilla/addons-server/pull/21966/files#diff-9d8c85c350642e45c3ffb7863b8d8a995351b815fb2a2db0dcfc6bb00748682eL103

This way we can push via the push_l10n_extraction.sh script.

I would merge this, make sure it's working. And then sometime early next week open a PR removing the extract-l10n.sh script as a redundant piece of code.

Reply:
oh, the `extract-l10.sh` script - I took "that script" to relate to the commented upon line - `push_l10n_extraction.sh`

Reply:
I try to avoid pronouns but everyone and a while I get lazy. Almost always immediately explodes. That should never be so vague. (see what I did there 😆 ) I'll resolve this (did it again)